dspace-api/src/test/java/org/dspace/AbstractIntegrationTestWithDatabase.java

```diff
@@ -206,6 +209,16 @@ public void destroy() throws Exception {
         }
     }
 
+    /**
+     * Clean up service caches after tests
+     * @param serviceManager
+     */
+    private void cleanupCache(ServiceManager serviceManager) {
+        MetadataAuthorityServiceImpl metadataAuthorityService = serviceManager
+            .getServiceByName(null, MetadataAuthorityServiceImpl.class);
+        metadataAuthorityService.clearCache();
```

Comment:
It seems odd to me that we'd need to have this cache cleared _after every test_ (in the destroy() method of this class).  I completely understand needing this cache to be cleared for Authority specific tests, like in the new code you've added to `DiscoveryRestControllerIT`.  Could you describe why this needs to be cleared *both* globally (for every IT test run) and within specific tests (like `DiscoveryRestControllerIT.discoverFacetsAuthorWithAuthorityWithSizeParameter()`? 

It seems like we should only need one or the other, so it's unclear why we need both....and if we don't need both, I'd prefer to remove this global `cleanupCache()` method in favor of clearing the cache only when necessary for the test.